*** configure.ac.current	2025-04-24 13:30:27.417316524 -0500
--- configure.ac	2025-04-24 13:31:15.170318835 -0500
*************** VISUALIZATION_PLUGINS=""
*** 81,91 ****
  CONTAINER_PLUGINS="asx asx3 audpl m3u pls xspf"
  TRANSPORT_PLUGINS="gio"
  
  if test "x$USE_GTK" = "xyes" ; then
      EFFECT_PLUGINS="$EFFECT_PLUGINS ladspa"
!     GENERAL_PLUGINS="$GENERAL_PLUGINS albumart lyrics-gtk playlist-manager search-tool statusicon"
      GENERAL_PLUGINS="$GENERAL_PLUGINS gtkui skins"
      VISUALIZATION_PLUGINS="$VISUALIZATION_PLUGINS blur_scope cairo-spectrum vumeter"
  fi
  
  if test "x$USE_QT" = "xyes" ; then
--- 81,91 ----
  CONTAINER_PLUGINS="asx asx3 audpl m3u pls xspf"
  TRANSPORT_PLUGINS="gio"
  
  if test "x$USE_GTK" = "xyes" ; then
      EFFECT_PLUGINS="$EFFECT_PLUGINS ladspa"
!     GENERAL_PLUGINS="$GENERAL_PLUGINS albumart lyrics-gtk playback-history-gtk playlist-manager search-tool statusicon"
      GENERAL_PLUGINS="$GENERAL_PLUGINS gtkui skins"
      VISUALIZATION_PLUGINS="$VISUALIZATION_PLUGINS blur_scope cairo-spectrum vumeter"
  fi
  
  if test "x$USE_QT" = "xyes" ; then
