*** menurow-current.cc	2025-04-29 17:03:24.956434946 -0500
--- menurow.cc	2025-04-29 17:04:59.502439522 -0500
***************
*** 23,32 ****
--- 23,36 ----
   *
   * The Audacious team does not consider modular code linking to
   * Audacious or using our public API to be a derived work.
   */
  
+ /* This code handles the old vertical menu-buttons:  [O A I D V] on
+  * left-side of the main window.
+  */
+ 
  #include <libaudcore/runtime.h>
  
  #include "skins_cfg.h"
  #include "skin.h"
  #include "menurow.h"
*************** void MenuRow::draw (cairo_t * cr)
*** 41,56 ****
              skin_draw_pixbuf (cr, SKIN_TITLEBAR, 312, 0, 0, 0, 8, 43);
      }
      else
          skin_draw_pixbuf (cr, SKIN_TITLEBAR, 304 + 8 * (m_selected - 1), 44, 0, 0, 8, 43);
  
      if (m_pushed)
      {
!         if (aud_get_bool ("skins", "always_on_top"))
!             skin_draw_pixbuf (cr, SKIN_TITLEBAR, 312, 54, 0, 10, 8, 8);
!         if (aud_get_bool ("skins", "double_size"))
!             skin_draw_pixbuf (cr, SKIN_TITLEBAR, 328, 70, 0, 26, 8, 8);
      }
  }
  
  static MenuRowItem menurow_find_selected (int x, int y)
  {
--- 45,71 ----
              skin_draw_pixbuf (cr, SKIN_TITLEBAR, 312, 0, 0, 0, 8, 43);
      }
      else
          skin_draw_pixbuf (cr, SKIN_TITLEBAR, 304 + 8 * (m_selected - 1), 44, 0, 0, 8, 43);
  
+     if (aud_get_bool ("skins", "always_on_top"))
+         skin_draw_pixbuf (cr, SKIN_TITLEBAR, 312, 54, 0, 10, 8, 8);
+     if (aud_get_bool ("skins", "double_size"))
+         skin_draw_pixbuf (cr, SKIN_TITLEBAR, 328, 70, 0, 26, 8, 8);
+ 
      if (m_pushed)
      {
!         if (m_selected == 2)
!         {
!             if (! aud_get_bool ("skins", "always_on_top"))
!                 skin_draw_pixbuf (cr, SKIN_TITLEBAR, 312, 54, 0, 10, 8, 8);
!         }
!         else if (m_selected == 4)
!         {
!             if (! aud_get_bool ("skins", "double_size"))
!                 skin_draw_pixbuf (cr, SKIN_TITLEBAR, 328, 70, 0, 26, 8, 8);
!         }
      }
  }
  
  static MenuRowItem menurow_find_selected (int x, int y)
  {
